packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -1384,7 +1381,6 @@ function mountIndeterminateComponent(
     }
 
     setIsRendering(true);
-    ReactCurrentOwner.current = workInProgress;
```

Comment:
This is probably unsafe and might break String Refs in Factory Components. Not sure if we support those.